content/toc-doc.json

```diff
@@ -103,6 +103,96 @@
       }
     ]
   },
+  {
```

Comment:
May want to put this further down in the sidebar nav. We currently have a "Going To Production" Nav Item. I wonder if it could make sense to either have this below that, or to have Tina Cloud & Self-hosted Tina be child siblings of that section.

Reply:
I will put it below it for now